backend/src/repository/getFaculties.js

```diff
@@ -0,0 +1,13 @@
+import connection from "../sqlConnection.js";
+
+export const getFaculties = async () => {
+  try {
+    const [results, fields] = await connection.execute(
+      `SELECT * FROM Faculties`
+    );
+
+    return results;
+  } catch (err) {
+    console.log(err);
```

Comment:
Kituose gražini false jei pavyksta, o čia nieko. Sakyčiau tiek čia, tiek kitur jei nepavyko reiktų throwint errorą. Tipo:
`throw new Error("Failed to get faculties")`